src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -223,6 +223,7 @@ class EmojiPickerMenu extends Component {
                     removeClippedSubviews={false}
                     contentContainerStyle={styles.flexGrow1}
                     ListEmptyComponent={<Text style={[styles.disabledText]}>{this.props.translate('common.noResultsFound')}</Text>}
+                    scrollEnabled={this.state.filteredEmojis.length !== 0}
```

Comment:
@GItGudRatio Can we use `alwaysBounceVertical` instead of scrollEnabled?

```suggestion
                    alwaysBounceVertical={this.state.filteredEmojis.length !== 0}
```